test/integration/combo-box-template.test.js

```diff
@@ -87,37 +86,4 @@ describe('item template', () => {
       });
     });
   });
-
-  describe('nested template', () => {
```

Comment:
Removing this test instead of fixing it as we'll also remove `polymer-legacy-adapter` in V25 anyway.